src/App.vue

```diff
@@ -1,24 +1,30 @@
 <template>
   <div id="app">
-    <building/>
-		<about/>
-		<gallery/>
-		<tail/>
+		<Navigation/>
+		<Home/>
+    <Building/>
+		<About/>
```

Comment:
I don't understand the spacing here. It looks like `About` `Gallery` and `Tail` are nested under `Building` if you look at the spacing, although they're not nested.

Reply:
Locally it isn't that way